sequence_python/models/ir_sequence.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2020 Sunflower IT (<https://sunflowerweb.nl>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+import random
+import string
+import uuid
+from inspect import getmembers, isclass, isfunction
+
+from odoo import fields, models
+from odoo.tools.barcode import get_barcode_check_digit
```

Comment:
This barcode thing seems to be something very specific. Maybe better put in a separate module....

Reply:
Added it to https://github.com/OCA/delivery-carrier/pull/902 instead, where I'm using it.